src/main/java/edu/iis/powp/events/predefine/SelectTestFigureOptionListener.java

```diff
@@ -16,6 +21,28 @@ public SelectTestFigureOptionListener(DriverManager driverManager) {
 
 	@Override
 	public void actionPerformed(ActionEvent e) {
+
+		Circle circle = new Circle(0,0,150);
+		Rectangle rectangle = new Rectangle(0,0,400,300);
+
+		switch (e.getActionCommand())
+        {
+		case "Figure Joe 1":
 		FiguresJoe.figureScript1(driverManager.getCurrentPlotter());
+
+        case "Figure Joe 2":

```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.